src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const headTableRow = thead.querySelector('tr');
+const tbody = document.querySelector('tbody');
+const tr = [...tbody.querySelectorAll('tr')];
+
+headTableRow.addEventListener('click', (e) => {
+  sortList(tr, [...headTableRow.children].indexOf(e.target));
```

Comment:
Hm, mb better to do this with `e.target.closet('th')`. So when you will receive `Name, Position, ...`. Your variant also can be but it's has a minus because `switch/case` has `0, 1` (not names) as cases.